container-engine-lib/lib/backend_impls/docker/docker_manager/create_and_start_container_args.go

```diff
@@ -249,3 +255,13 @@ func (builder *CreateAndStartContainerArgsBuilder) WithContainerInitEnabled(cont
 	builder.containerInitEnabled = containerInitEnabled
 	return builder
 }
+
+func (builder *CreateAndStartContainerArgsBuilder) WithPullingLatestImageAlways() *CreateAndStartContainerArgsBuilder {
+	builder.imageDownloadMode = image_download_mode.Always
+	return builder
+}
+
+func (builder *CreateAndStartContainerArgsBuilder) WithPullingLatestImageOnMissing() *CreateAndStartContainerArgsBuilder {
```

Comment:
could we change this to `WithPullingLatestImageIfMissing` , I think `s/On/If` will help express the behavior is conditional.